src/pages/DataUsage/index.tsx

```diff
@@ -145,23 +180,37 @@ export const DataUsagePage: React.FC = () => {
                   type="date"
                   label="Select Start Date"
                   placeholder="From"
-                  {...formik.getFieldProps('startDate')}
+                  name="startDate"
+                  value={minDate}
+                  onChange={(e) => {
+                    setMinDate(e.target.value);
+                    formik.setFieldValue('startDate', e.target.value);
+                  }}
                   error={getFieldError(
                     formik.errors.startDate,
                     formik.touched.startDate,
                   )}
+                  min={minDate}
+                  disabled={loading}
                 />
               </Column>
               <Column useAppMargin md={4} lg={3}>
                 <TextField
                   type="date"
                   label="Select End Date"
                   placeholder="To"
-                  {...formik.getFieldProps('endDate')}
+                  name="endDate"
+                  value={maxDate}
+                  onChange={(e) => {
+                    setMaxDate(e.target.value);
+                    formik.setFieldValue('endDate', e.target.value);
+                  }}
                   error={getFieldError(
                     formik.errors.endDate,
                     formik.touched.endDate,
                   )}
+                  min={maxDate}
```

Comment:
Don't know if this was intentional but should the min also be maxDate?

Reply:
This has been removed. I was trying to see if i could limit the dates the user can select.